pkg/review/commit_review_status.go

```diff
@@ -167,16 +183,51 @@ type CommitApprovalPipelineConfig struct {
 // CommitApprovalDoFn is an object that implements beams "DoFn" interface to
 // provide the processing logic for converting a Commit to CommitReviewStatus.
 type CommitApprovalDoFn struct {
-	Config       CommitApprovalPipelineConfig
-	GithubClient *githubv4.Client
+	Config CommitApprovalPipelineConfig
```

Comment:
Can this be lower "c" `config`? No need to expose it if not necessary.
Can this be a pointer? `*CommitApprovalPipelineConfig`? More of a convention - use pointer whenever possible

Reply:
it needs to be public so that it can be serialized. Apache Beam will only serialize public field of a struct. For similar reasons I don't think using a pointer makes sense here as I don't think beam will serialize it correctly if it's a pointer.

Reply:
Curious - Does it require the `Config` being public or the just the fields in it? 

Could you add a short comment above the config about serializing? I know you've documented somewhere else but I think it's better to stay closer to the thing to be serialized.